be/src/main/java/yeonba/be/mypage/dto/request/UserUpdateUnwantedAcquaintanceRequest.java

```diff
@@ -4,23 +4,17 @@
 import jakarta.validation.constraints.Pattern;
 import lombok.AllArgsConstructor;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
 
 @Getter
+@NoArgsConstructor
 @AllArgsConstructor
```

Comment:
```suggestion
@AllArgsConstructor
```
request 객체에 AllArgsConstructor가 필요한 이유를 말씀해주시면 감사하겠습니다!

Reply:
변경 내용 보시면 제가 이번 PR에서 추가한 게 아닌 것같은데용? 불필요한 것같으니까 삭제하겠습니다.